ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -560,6 +560,44 @@ def test_paddle_tensor_argmax(
     )
 
 
+# lerp
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="lerp",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), num_arrays=2, shared_dtype=True
```

Comment:
Best to use `helpers.get_dtypes("valid")` here and keep the supported dtypes inside the `with_supported_dtypes`1 decorator applied to the `lerp` method in the `Tensor` class